modules/custom/cu_core/cu_core.module

```diff
@@ -4,6 +4,15 @@
  * Contains hooks for cu_core module.
  */
 
+/**
+ * Implements hook_module_implements_alter().
+ */
+function cu_sitemap_menu_alter(&$items) {
+  // This is a place to remove visibility for enpoints
+  // unnecessarily set as public.
+  $items['filter/tips']['access callback'] = FALSE;
+}
```

Comment:
Should be in the cu_sitemap module

Reply:
Why in the cu_sitemap module? I do not think that this bug is actually related to that module.

Reply:
Based on the name of your hook, traditionally you swap the word `hook` for the name of the module that is implementing this. I would either move the function or rename it to `cu_core_menu_alter`

Reply:
Oh, of course. That was a typo! Sorry. I meant to implement it in cu_core as cu_core_menu_alter(). I must have forgot to commit my final changes and push those up.